react-components/table-intro/step2.md

```diff
@@ -1,12 +1,12 @@
 Let's create the basic table structure.
 
-## Task
-
-1) Navigate to the `src` folder and open `src/App.js`{{open}}
+1) <strong>Navigate to the `src` folder and open `src/App.js`.</strong>
 
 Next, we'll set up the general structure of an empty table and add the necessary imports.
```

Comment:
Remove instances of "we'll" - change to `Set up the general structure of an empty table and add the necessary imports.`